.ci/test/behat/cleanup

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -eo pipefail
+
+#
+# This script runs cleanup after Behat tests complete.
+#
+
+# Authenticate with Terminus
+terminus -n auth:login --machine-token="$TERMINUS_TOKEN"
+
+# Restore the DB backup made before testing
+terminus -n backup:restore $TERMINUS_SITE.$TERMINUS_ENV --element=database --yes
+
+# Clear Drupal cache
+terminus -n drush "$TERMINUS_SITE.$TERMINUS_ENV" cr
+
+# Clear site cache
+terminus -n env:clear-cache $TERMINUS_SITE.$TERMINUS_ENV
+
+# Store artifacts
+export ARTIFACTS_DIR_URL="${CI_BUILD_URL}/artifacts/${CI_NODE_INDEX}/artifacts"
+
+# Create the artifacts directory if needed
+if [ ! -d "$ARTIFACTS_FULL_DIR" ]; then
+  mkdir -p $ARTIFACTS_FULL_DIR
+fi
+
+
+# Copy behat-screenshots files to ARTIFACTS_FULL_DIR
```

Comment:
This is awesome.

Reply:
I've had issues reconfiguring the directory [Behat fail aid](https://packagist.org/packages/genesis/behat-fail-aid) saves screenshots to but I was successful in [writing my own screenshot step](https://github.com/ataylorme/ataylorme-wordpress/blob/master/tests/behat/helper_classes/Contexts/MiscContext.php#L38:L58) using the Chrome Driver that works well. Think that is worth adding here?

I could use it in a few places in the existing example feature file, e.g. when created articles, users or terms are viewed in the admin.